sources/command/boot.py

```diff
@@ -3,38 +3,48 @@
 
 from globals import ExitCode, get_devices
 from systemd import Systemd
+from openrc import Openrc
```

Comment:
Importing in this way will cause the tool to crash on all distributions that do not use openrc.

Reply:
I will fix it.